corgi/core/models.py

```diff
@@ -533,6 +530,8 @@ def get_latest_components(self):
                         root_components,
                         name=OuterRef("name"),
                         product_streams__overlap=(self.ofuri,),
+                        # TODO: What is overlap for??
+                        # TODO: Exclude software_build__isnull=True???
```

Comment:
Not a bad idea. Although I don't think there will be any of these as the root components types should always have software_build set. There is no DB constraint for that however so if it was true, it would be a bug in the way those components were created.